src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_SIZE = 10;
+    private K[] keyArray = (K[]) new Object[MAX_ARRAY_SIZE];
+    private V[] valueArray = (V[]) new Object[MAX_ARRAY_SIZE];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class fields have default values